### ISSUE TITLE
Ctrl + Alt + number keys not working in 1.7.1033.0 with Danish keyboard

### ISSUE THREAD

**Issue by oback** [microsoft/terminal#9909]
On Danish layout keyboards several characters are accessed through the `Alt Gr` or `Ctrl+Alt` modifier keys (either will usually work). For instance, `[` is typed with `AltGr + 8` or `Ctrl + Alt + 8`. Due to the various placement of the keys, it's nice to have both options, since for example `AltGr + 8` can be awkward to reach.

After wt was upgraded to 1.7.1033.0, the `Ctrl + Alt` combinations no longer work. `AltGr` still works, and the `Ctrl + Alt` options also work directly in cmd.exe and powershell terminals.

If I uninstall the latest version and manually install version 1.6.10571.0, the `Ctrl + Alt` modifiers work again.

Configuration:
Windows Terminal 1.7.1033.0
Danish QWERTY keyboard layout

**Comment by oback** [comment]
After looking into it some more, I realized that those key combinations are bound to tab switching. If I unbind the keys I get the old behavior back:

```
        { "command": "unbound", "keys": "ctrl+alt+1" },
        { "command": "unbound", "keys": "ctrl+alt+2" },
        { "command": "unbound", "keys": "ctrl+alt+3" },
        { "command": "unbound", "keys": "ctrl+alt+4" },
        { "command": "unbound", "keys": "ctrl+alt+5" },
        { "command": "unbound", "keys": "ctrl+alt+6" },
        { "command": "unbound", "keys": "ctrl+alt+7" },
        { "command": "unbound", "keys": "ctrl+alt+8" },
        { "command": "unbound", "keys": "ctrl+alt+9" }
```

As far as I can tell the same key bindings existed in the previous version. I'm not sure what has changed, but I have a reasonable workaround for now.

**Comment by zadjii-msft** [comment]
Huh. That's weird. I don't _think_ we [merged anything into 1.7](https://github.com/microsoft/terminal/compare/v1.7.1033.0...release-1.6) that had anything to do with the input handling, especially not altgr handling. 91f68e2914dbdde338c58e0ab4095bee49d1470d is the closest to touching that, and I don't think that should have broke it. 

@lhecker you want to give this a look?

**Comment by oback** [comment]
> Huh. That's weird. I don't _think_ we [merged anything into 1.7](https://github.com/microsoft/terminal/compare/v1.7.1033.0...release-1.6) that had anything to do with the input handling, especially not altgr handling. 

The only other version I tested was 1.6.10571.0, and the [diff between that and 1.7.1033.0](https://github.com/microsoft/terminal/compare/v1.6.10571.0...v1.7.1033.0) seems to be a bit bigger - in particular it includes #5272 and #8458.

But I would almost say the current behavior is actually correct. The `ctrl+alt+<number>` inputs are now hidden by (default) key bindings, but from what I can tell this is the desired behavior. If I bind a key to another normal input (like `{ "command": "closeWindow", "keys": "a" },` for an extreme example), then the command fires instead of the input. For some reason `ctrl + alt + <number>` gave priority to the input over the command binding in previous versions, whereas it is now in line with other inputs.

**Comment by lhecker** [comment]
I've installed the "Danish" language and keyboard layout, as well as version 1.6.10571.0 of Windows Terminal.

Unfortunately I was unable to reproduce the old behavior you describe: Even in version 1.6 `Ctrl+Alt+<num>` will switch tabs for me instead of working as an alias for AltGr.

> But I would almost say the current behavior is actually correct.

That's right and the "unbound" commands your posted earlier, are the correct approach to restore the Ctrl+Alt behavior you expect. The fact that `Ctrl+Alt+<num>` is used to switch tabs, might be considered an unfortunate choice from the past, but we currently stick with it, in order to not break the workflow for users who're already used to that.

And due to that I'd like to take the liberty to close this issue for now. 🙂
